lab-gurpreet/readme.md

```diff
@@ -0,0 +1,9 @@
+as a developer
+    You need to install node package and its dependencies globally.
```

Comment:
So, with node projects, assuming that your package.json file has all of the necessary dependencies, all you need to do is instruct the developer to run 'npm install' on the command line. Since you also have a gulp file, you could also instruct the developer to run linting and tests through gulp rather than requiring the developer to install eslint and mocha globally.